bitrise.yml

```diff
@@ -160,7 +160,17 @@ workflows:
             #!/usr/bin/env bash
             set -ex
 
-            bash <(curl -s https://codecov.io/bash) -J '^RInAppMessaging$' -X gcov
+            curl -sS https://codecov.io/bash > codecov
+
+            # verify checksum
+            VERSION=$(grep -o 'VERSION=\"[0-9\.]*\"' codecov | cut -d'"' -f2)
+            for i in 1 256 512
```

Comment:
Why check all these algo variants?

Reply:
I'm not sure to be honest. Just in case?
I followed the example in Codecov docs https://docs.codecov.io/docs/about-the-codecov-bash-uploader#validating-the-bash-script (that example doesn't work as is)